Move all the examples into different repos, so that leave in this rep… (#68)
Kotlin/kotlin-wasm-examples: README.md

These are the Kotlin/Wasm examples you can try:

> * **[Compose example](https://github.com/Kotlin/kotlin-wasm-compose-template/blob/main/README.md)** — A simple Compose Multiplatform application template

Reviewer: ```suggestion
* **[Compose example](https://github.com/Kotlin/kotlin-wasm-compose-template/#readme)** — A simple Compose Multiplatform application template
```

> * **[Compose Jetsnack](https://github.com/JetBrains/compose-multiplatform/blob/master/examples/jetsnack/README.md)** — Web version of a [Jetsnack application](https://github.com/android/compose-samples/tree/main/Jetsnack) built with Compose Multiplatform

Reviewer: ```suggestion
* **[Compose Jetsnack](https://github.com/JetBrains/compose-multiplatform/tree/master/examples/jetsnack/#readme)** — Web version of a [Jetsnack application](https://github.com/android/compose-samples/tree/main/Jetsnack) built with Compose Multiplatform
```

Reviewer: etc..

> * **[Compose image viewer](https://github.com/JetBrains/compose-multiplatform/blob/master/examples/imageviewer/README.md)** — Web version of an image viewer application built with Compose Multiplatform

Reviewer: ```suggestion
* **[Compose image viewer](https://github.com/JetBrains/compose-multiplatform/tree/master/examples/imageviewer/#readme)** — Web version of an image viewer application built with Compose Multiplatform
```

> * **[Browser example](https://github.com/Kotlin/kotlin-wasm-browser-template/blob/main/README.md)** — A simple application showing "Hello World" in the browser using DOM API

Reviewer: ```suggestion
* **[Browser example](https://github.com/Kotlin/kotlin-wasm-browser-template/#readme)** — A simple application showing "Hello World" in the browser using DOM API
```

> * **[Node.js example](https://github.com/Kotlin/kotlin-wasm-nodejs-template/blob/main/README.md)** — A simple application printing "Hello World" in Node.js

Reviewer: ```suggestion
* **[Node.js example](https://github.com/Kotlin/kotlin-wasm-nodejs-template/#readme)** — A simple application printing "Hello World" in Node.js
```

> * **[KoWasm](https://github.com/kowasm/kowasm)** — Server-side and full stack development with Kotlin and WebAssembly leveraging WASI and Component Model

Reviewer: ```suggestion
* **[KoWasm](https://github.com/kowasm/kowasm/#readme)** — Server-side and full stack development with Kotlin and WebAssembly leveraging WASI and Component Model
```

> * **[WASI example](https://github.com/Kotlin/kotlin-wasm-wasi-template/blob/main/README.md)** — A simple application using WASI API in Node.js and Deno

Reviewer: ```suggestion
* **[WASI example](https://github.com/Kotlin/kotlin-wasm-wasi-template/#readme)** — A simple application using WASI API in Node.js and Deno
```
